types/node/timers.d.ts

```diff
@@ -40,18 +40,27 @@ declare module 'timers' {
         }
 
         function setTimeout<TArgs extends any[]>(callback: (...args: TArgs) => void, ms?: number, ...args: TArgs): NodeJS.Timeout;
+        // util.promisify no rest args compability
+        // tslint:disable-next-line void-return
+        function setTimeout(callback: (args: void) => void): NodeJS.Timeout;
```

Comment:
Sorry, I know this PR has already been merged, but shouldn't this have the `ms` parameters as for `setInterval`?
I still have the `Expected 3 arguments, but got 2` error for `setTimeout` but not for `setInterval`

Reply:
You're right. Good catch. CC @birtles 

@SimonSchick Seems like tests are passing because of "dom" lib in `tsconfig.json`. How to test this properly?

Reply:
It's kinda tricky, in CI we need to test it with the dom lib to ensure compatibility, you can always temporarily remove it test, add it back, I understand this is a tiny bit of churn but should be OK for now.